apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -136,31 +140,36 @@ public void Dispose()
                                             Key                STRING(MAX) NOT NULL,
                                             StringValue        STRING(MAX),
                                           ) PRIMARY KEY (Key)";
+            string createDataTableStatement = $@"CREATE TABLE {DataAdapterTestTable} (
```

Comment:
they can definitely diverge as they are for a set of completely different test cases.
eg, its possible we want to test some unique type conversion scenarios support with datatables.